tutorials/cyber/network_mapping/lanl_network_mapping_using_rapids.ipynb

```diff
@@ -0,0 +1,758 @@
+   "metadata": {},
+   "source": [
+    "### Add numeric src and dst node ids to edge list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add contiguous src id's to edges\n",
+    "edges_gdf['id'] = edges_gdf['src']\n",
+    "edges_gdf = edges_gdf.merge(idmap_gdf, on=['id'])\n",
+    "edges_gdf['src_idx'] = edges_gdf['idx']\n",
+    "edges_gdf = edges_gdf.drop(['id', 'idx'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add contiguous dst id's to edges\n",
+    "edges_gdf['id'] = edges_gdf['dst']\n",
+    "edges_gdf = edges_gdf.merge(idmap_gdf, on=['id'])\n",
+    "edges_gdf['dst_idx'] = edges_gdf['idx']\n",
+    "edges_gdf = edges_gdf.drop(['id', 'idx'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create input edge list for cuGraph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cg_edges_gdf = edges_gdf[['src_idx', 'dst_idx']]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cg_edges_gdf['src_idx'] = cg_edges_gdf['src_idx'].astype('int32')\n",
+    "cg_edges_gdf['dst_idx'] = cg_edges_gdf['dst_idx'].astype('int32')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run cuGraph PageRank\n",
+    "\n",
+    "Next we create our graph and run pagerank."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "G = cugraph.Graph()\n",
+    "G.add_edge_list(cg_edges_gdf['src_idx'], cg_edges_gdf['dst_idx'], None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%time pr_gdf = cugraph.pagerank(G, alpha=0.85, max_iter=500, tol=1.0e-05)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(pr_gdf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add PageRank scores to node list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pr_gdf['idx'] = pr_gdf['vertex'].astype('int64')\n",
+    "all_nodes_gdf = all_nodes_gdf.merge(pr_gdf, on=['idx'], how='left')\n",
+    "all_nodes_gdf = all_nodes_gdf.drop(['vertex'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Graphistry Viz\n",
+    "\n",
+    "We use [Graphistry](https://www.graphistry.com/) to visualize the network mapping graph. A snapshot of the graph constructed from this notebook can be viewed below. Run the code below to contruct a graph using Graphistry.\n",
```

Comment:
@brhodes10 should we say something like:

"A snapshot of the graph constructed from this notebook is provided below. To generate it yourself, you'll need to register an account with Graphistry and configure your key below" ?

Reply:
Thanks Randy I have added that line